src/axom/primal/geometry/Polyhedron.hpp

```diff
@@ -222,14 +225,14 @@ class NeighborCollection
  *
  *       <pre>
  *
- *          4--------7
- *         /|       /|
- *        / |      / |
- *       5--------6  |
- *       |  0-----|--3
- *       | /      | /
+ *          4--------5          +y
```

Comment:
Again, left-handed coordinate system. Is this intended?

Reply:
@rhornung67 Yes, it is a left-handed coordinate system.